docs/sections/fundamentals/basics.md

```diff
@@ -249,65 +257,144 @@ As a final step, let's have a look at the provenance of this simple calculation.
-To do so, if you are using Linux/Mac OS X, you can type in your *local* machine:
+## CalcJobs
+
+When running calculations that require an external code or run on a remote machine, a simple calculation function is no longer sufficient.
+For this purpose, AiiDA provides the `CalcJob` process class.
+
+To see all calculations available from your AiiDA packages and get information about them you can use the `verdi plugin` command.
+This tool is used for all plugin, so running `verdi plugin list` will just list all possible groups of plugins.
 
 ```{code-block} console
 
-$ scp aiidatutorial:<path_to_the_graph_pdf> <local_folder>
+$ verdi plugin list
+Info: Available entry point groups:
+* aiida.calculations
+* aiida.cmdline.computer.configure
+* aiida.cmdline.data
+* aiida.cmdline.data.structure.import
+* aiida.data
+* aiida.groups
+* aiida.node
+* aiida.parsers
+* aiida.schedulers
+* aiida.tools.calculations
+* aiida.tools.data.orbitals
+* aiida.tools.dbexporters
+* aiida.tools.dbimporters
+* aiida.transports
+* aiida.workflows
+
+Info: Pass one of the groups as an additional argument to show the registered plugins
 
 ```
 
-and then open the file.
+If you want to list calculation in particular you can follow the indications present at the end of the output: specify the `aiida.calculations` group as an argument of the command.
+If you do, you will get a long list of *entry points* (the official label that univocally identifies a plugin within AiiDA), among which you should be able to see the `arithmetic.add`:
 
-```{note}
+```{code-block} console
 
-You can also use the `jupyter notebook` setup explained (**TODO: FIX LINK**) to download files.
-Note that while Firefox will display the PDF directly in the browser [Chrome and Safari block viewing PDFs from jupyter notebook servers](<https://stackoverflow.com/a/55264795/1069467>) - with these browsers, you will need to tick the checkbox next to the PDF and download the file.
+$ verdi plugin list aiida.calculations
+Registered entry points for aiida.calculations:
+(...)
+* arithmetic.add
+(...)
 
-Alternatively, you can use graphical software to achieve the same, for instance: on Windows: WinSCP; on a Mac: Cyberduck; on Linux Ubuntu: using the 'Connect to server' option in the main menu after clicking on the desktop.
+Info: Pass the entry point as an argument to display detailed information
 
 ```
 
-It should look something like the graph shown in {numref}`fig-calcfun-graph`.
+The `arithmetic.add` shown explicitly above is the `CalcJob` we will be using in what follows.
 
-(fig-calcfun-graph)=
+To get more information about the inputs, outputs, etc. of this calculation, we can just do as indicated again and pass the specific entry point as an additional argument for the command:
 
-```{figure} include/images/basics_calcfun_graph.png
-:scale: 50
-:align: center
+```{code-block} console
 
-Provenance graph of the `multiply` calculation function.
+$ verdi plugin list aiida.calculations arithmetic.add
```

Comment:
Interesting that this actually doesn't contain the _class name_ of the calculation job. Makes it more difficult to introduce, especially since the `CalculationFactory` doesn't just load that class or give you any information on the class name?